Formula/crackpkcs.rb

```diff
@@ -0,0 +1,23 @@
+class Crackpkcs < Formula
+  desc "Multithreaded program to crack PKCS#12 files"
+  homepage "https://crackpkcs12.sourceforge.io"
+  url "https://download.sourceforge.net/project/crackpkcs12/0.2.11/crackpkcs12-0.2.11.tar.gz"
+  sha256 "9cfd0aa1160545810404fff60234c7b6372ce7fcf9df392a7944366cae3fbf25"
+  license "GPL-3.0-or-later"
+
+  depends_on "openssl@1.1"
+  depends_on "pkg-config"
+
+  def install
+    system "./configure",
+            *std_configure_args,
+            "--disable-silent-rules",
+            "--with-openssl=#{Formula["openssl@1.1"].opt_prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "curl", "-o", "cert.p12", "-L", "https://github.com/crackpkcs12/crackpkcs12/blob/master/misc/utils/certs/usr0052-exportado_desde_firefox.p12?raw=true"
+    system "#{bin}/crackpkcs12", "-m", "7", "-M", "7", "-s", "usr0052", "-b", "cert.p12"
```

Comment:
Is there any output here that we can check?

Reply:
It looks like 

```
Brute force attack - Starting 8 threads

Alphabet: usr0052
Min length: 7
Max length: 7
Use -m and -M flags to modify these values.

**********************************************************
Brute force attack - Thread 1 - Password found: usr0052
**********************************************************
```

Reply:
Then we can use `shell_output` instead of `system` and check that string `Brute force attack - Thread 1 - Password found: usr0052` appears in that output. It's going to be something like this
```ruby
output = shell_output("#{bin}/crackpkcs12 -m 7 -M 7 -s usr0052 -b cert.p12")
assert_match "Brute force attack - Thread 1 - Password found: usr0052", output
```

Reply:
ok, thanks.